clang/lib/CIR/Dialect/Transforms/TargetLowering/RecordLayoutBuilder.cpp

```diff
@@ -223,6 +223,10 @@ class ItaniumRecordLayoutBuilder {
   /// Initialize record layout for the given record decl.
   void initializeLayout(const Type Ty);
 
+  /// FinishLayout - Finalize record layout. Adjust record size based on the
```

Comment:
Remove the function name from comment " FinishLayout -"

Reply:
updated)